static/js/containers/pages/ReceiptPage.js

```diff
@@ -90,6 +90,7 @@ export class ReceiptPage extends React.Component<Props> {
                 </div>
               </div>
             )}
+            {currentUser.is_authenticated}
```

Comment:
Is that intentional to use it twice as it's being used in the next line as well?

Reply:
Good catch. added for testing .